pkg/provider/github/events.go

```diff
@@ -37,6 +43,59 @@ func (v *Provider) payloadFix(payload string) []byte {
 	return []byte(replacer.Replace(payload))
 }
 
+func (v *Provider) fetchAppToken(ctx context.Context, kube kubernetes.Interface, info *info.PacOpts, installationID int64) error {
+	// TODO: move this out of here
```

Comment:
we maybe would want to have some caching since his is going to be fetched  on every gapp event

Reply:
yeah we might to use informer for secret.. using api call will hit the api server limit 
I will add a todo